api/views.py

```diff
@@ -12,3 +13,14 @@ class UserViewSet(ModelViewSet):
     queryset = User.objects.all()
     permission_classes = (IsAuthenticated, IsAdminUser)
     http_method_names = ['get', 'post']
+
+
+class ItemViewSet(ModelViewSet):
+    serializer_class = ItemSerializer
+    permission_classes = [IsAuthenticated, ]
+    http_method_names = ['get']
```

Comment:
Does the `delete`, `patch` and `put` need to be here as you have endpoints for those http verbs ?

Reply:
@HawiCaesar The story limits only `list` for the normal user. Admin user may be allowed to `delete`, `patch` and `put`